src/Entities/Nodes.jl

```diff
@@ -65,17 +65,8 @@ dofs(vn::Vector{<:AbstractNode}) = vcat(dofs.(vn)...) # mapreduce(dofs, vcat, vn
 dofs(n::AbstractNode, s::Field) = n.dofs[s]
 
 "Sets a vector of dofs `vd` to the node assigned to the field `s`."
-function apply!(n::AbstractNode, s::Field, vd::Vector{Dof})
-    if s ∉ keys(dofs(n))
-        insert!(dofs(n), s, vd)
-    else
-        for d in vd
-            if d ∉ dofs(n)[s]
-                push!(dofs(n)[s], d)
-            end
-        end
-    end
-    dofs(n)
+function set_dofs!(n::AbstractNode, s::Field, vd::Vector{Dof})
+    unique!(append!(get!(dofs(n), s, Dof[]), vd))
```

Comment:
Took the opportunity to reduce this function a bit; might do something along these lines for the other method dispatching on `Mesh` on a separate PR.

Reply:
cf https://github.com/ONSAS/ONSAS.jl/issues/362